lib/scope.js

```diff
@@ -518,30 +506,30 @@ module.exports = {
     let page_data = { sought_var: '', fields: [] };
     const $ = cheerio.load( html );  // Could this be on `scope`?
 
-    // All these pieces of data feel very divorced from the field itself and the action happening.
-    // Surely there's a better way.
-
     // Get sought var
-    let encoded_sought_var = $($( `input[name="_event"]` )).attr( `value` );
-    // TODO: Change to `sought_name`?
-    let sought_var_str = await scope.fromBase64( scope, { base64_str: encoded_sought_var });
-    page_data.sought_var = JSON.parse( sought_var_str )[0];  // Why is this a list? Can multiple vars be sought?
+    let encoded_sought_var = $($( `#sought_variable` )).data( `variable` );  // This should come out to 'None' at the very least
+    // TODO: Change to `sought_name` or `trigger`?
+    page_data.sought_var = await scope.fromBase64( scope, { base64_str: encoded_sought_var });
+    if ( !page_data.sought_var ) {
+      // TODO: Add documentation and then link to it in this message.
+      await scope.addToReport(scope, { key: 'ids', value: `Warning: You are missing an element containing the data for the page's sought/trigger variable. Some fields, like loops with index variables, may not work correctly in these tests. Talk to the folks at the AssemblyLine project to learn more.` });
```

Comment:
This gets added to the report for every page, but I'm not sweating the small stuff right now.

Reply:
Made #260